azure-pipelines.yml

```
@@ -1 +1,84 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
 
+pr: none
+
+variables:
+  - group: 'Prod Management Resources'
+  - name: buildConfiguration
+    value: 'release'
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: "GitHub (SFA)"
+
+jobs:
+- job: 'CodeBuild'
+  pool:
+    name: 'DAS - Continuous Integration'
+    demands: Agent.OS -equals Linux
+  workspace:
+    clean: all  
+  steps:
+  - task: gittools.gitversion-preview.gitversion-task.GitVersion@5
```

```suggestion
  - task: UseGitVersion@5
```
Latest version of task that is installed on the organisation